Packs/ProactiveThreatHunting/IncidentTypes/incidenttype-Proactive_Threat_Hunting.json

```diff
@@ -0,0 +1,28 @@
+{
+    "autorun": true,
+    "color": "#F8E7A5",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
```

Comment:
Maybe add some indicator extraction rules so that when fields are changed indicators will be extracted